call-home/src/collector/http_client.rs

```diff
@@ -0,0 +1,92 @@
+use std::time::Duration;
```

Comment:
if we use the openapi-generator (https://github.com/openebs/mayastor-control-plane/tree/develop/nix/pkgs/openapi-generator) we get all this for free